docs/source/architecture/journal.rst

```diff
@@ -394,84 +407,96 @@ The file contains a protobuf-encoded list of batches:
-the consensus mechanism. The second block will then use the configured
-consensus model, which will need to know how to initialize the consensus field
-from an empty one.  In future cases, transitions between consensus models may be
-possible, as long as they know how to read the consensus field of the previous
-block.
-
-To complete the process, all necessary transaction processors must be running.
-A minimum requirement is the Sawtooth Settings transaction processor,
-``settings-tp``.
+          sawtooth.poet.initial_wait_timer={value} \
+          sawtooth.poet.target_wait_time={value} \
+          sawtooth.poet.population_estimate_sample_size={value}
+
+	$ sawadm genesis sawset.batch
+
+When ``sawadm genesis`` runs, it writes a ``genesis.batch`` file to the
+validator's data directory (for more information, see
+:doc:`../cli/sawadm` in the CLI Command Reference).
+
+.. note::
+
+   For a network using PoET consensus, additional batches must be included in
+   the genesis batch. For more information, see :ref:`proc-multi-ubuntu-label`
+   in the Application Developer's Guide.
+
+
+Genesis Block Creation
+----------------------
+
+Once the genesis batch exists, the validator uses the following process to
+produce a genesis block.
+
+1. When a validator starts, it determines that it must produce a genesis block
+   if the following conditions are true:
+
+   * The ``genesis.batch`` file exists
+   * There is no block specified as the chain head
+
+   If either of these conditions is not met, the validator halts operation.
+
+2. The validator loads the batches from the ``genesis.batch`` file into the
+   pending queue.
+
+3. The validator produces the genesis block using the standard block-creation
+   process, with the following modifications:
+
+   * The batch execution order is strictly in the order of the batches in the
+     ``GenesisData`` list. The :ref:`txn-sched-executor-label` does not attempt
+     to reorder batches or drop failed transactions. If any transaction in
+     ``genesis.batch`` fails, no genesis block is produced. The validator
+     treats this failure as a fatal error.
+
+   * The validator determines block validity without considering consensus.
+     At the start of the genesis block creation process, state (the
+     Merkle-Radix tree) is empty, so the blockchain does not have a consensus
+     algorithm yet. As a result, the genesis block has an empty consensus field.
+
+4. The validator also writes the blockchain ID (that is, the signature of the
+   genesis block) to the file ``block-chain-id`` in the validator’s data
+   directory. The blockchain ID is used to mark this validator's genesis block
+   as the correct one for the blockchain.
+
+Genesis Block Processing
+------------------------
+
+To complete the genesis operation process and commit the genesis block, all
+necessary transaction processors must be running. At a minimum, the Settings
+transaction processor (:doc:`../cli/settings-tp`) is required.
```

Comment:
This isn't true. You can deploy a validator without a Settings transaction processor if no settings are being set or you could deploy it with your own implementation of a transaction processor to handle on-chain settings. I would change "At a minimum, the Settings transaction processor (:doc:`../cli/settings-tp`) is required." to "If any transactions in the genesis block modify settings, the Settings transaction processor (:doc:`../cli/settings-tp`) or an alternative implementation should be among those started."